IHP/ViewSupport.hs

```diff
@@ -94,7 +95,7 @@ currentViewId =
         moduleParts :: [Text]
         moduleParts = Text.splitOn "." moduleName
 
--- | Returns @True@ when the current request path matches the path given as argument.
+-- | Returns @True@ when the current request path matches the path given as argument. Takes into account the search query: `?name=value`
```

Comment:
In haskell documentation you have to use `@` symbols for inline code like `@?name=value`